Documentation/Development.md

```diff
@@ -54,6 +54,9 @@ therefore build a new version.
 
 #### Building
 
+<details>
+    <summary>About the Scripts</summary>
+
 The X10 implementation is distributed as part of the Swift for TensorFlow APIs
```

Comment:
This portion isn't about the build scripts but rather the tree structure and components.